source/framework/vx_graph_verify.c

```diff
@@ -2044,7 +2091,7 @@ VX_API_ENTRY vx_status VX_API_CALL vxVerifyGraph(vx_graph graph)
             if(status == (vx_status)VX_SUCCESS)
             {
                 /* Call target kernel init for each node
-                 * This results in message coomunication with target kernels
+                 * This results in message communication with target kernels
```

Comment:
This should also be fixed in the latest set of changes

Reply:
to be removed from the PR @RB